internal/buildscripts/packaging/tests/installer_test.py

```diff
@@ -206,7 +206,7 @@ def fluentd_supported(distro, arch):
         return False
     elif distro == "amazonlinux-2023":
         return False
-    elif distro in ("debian-stretch", "ubuntu-xenial") and arch == "arm64":
+    elif distro in ("debian-stretch", "ubuntu-xenial", "ubuntu-noble") and arch == "arm64":
```

Comment:
```suggestion
    elif distro in ("debian-stretch", "ubuntu-xenial") and arch == "arm64":
        return False
    elif distro == "ubuntu-noble":
        return False
```
We want to skip fluentd tests for ubuntu-noble, regardless of arch.